x-pack/plugins/lens/public/data_views_service/loader.ts

```diff
@@ -0,0 +1,310 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { isNestedField } from '@kbn/data-views-plugin/common';
```

Comment:
data views service _and_ index pattern service? Seems like a bit much. Did you rename and a merge conflict got resolved this way?